lib/SSH/ssh.cpp

```diff
@@ -0,0 +1,144 @@
+#include "ssh.h"
+
+#include <fstream>
+#include <iostream>
+#include <memory>
+#include <string>
+
+int SSH::fileTransfer(std::string localFile, std::string serverPath) {
+    ssh_session session = createSession();
+
+    if (session == nullptr) {
+        std::cout << "Session is null";
+        return -1;
+    }
+
+    std::ifstream file(localFile, std::ios::binary);
+    if (not file.is_open()) {
+        std::cout << "Error opening file";
```

Comment:
```suggestion
        std::cout << "Unable to access file for file transfer";
```

Also print the file name

Reply:
applied